docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -1,12 +1,12 @@
 [id="migrating-the-rgw-backends_{context}"]
 
-= Migrating the {Ceph} RGW backends
+= Migrating the {Ceph} RGW back ends
 
-To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
+You must migrate your Ceph Object Gateway (RGW) back ends from your Controller nodes to your {Ceph} nodes. To ensure that you distribute the correct number of services to your available nodes, you use `cephadm` labels to refer to a group of nodes where a given daemon type is deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
```

Comment:
I might say "amount" instead of "number" in case "number" gets translated like a registration number or something like that